deploy/helm/templates/rbac.yaml

```diff
@@ -141,13 +141,21 @@ rules:
       - configauditreports
       - clusterconfigauditreports
       - ciskubebenchreports
+      - clustercompliancereports
```

Comment:
We need to update deploy/static/02-starboard-operator.rbac.yaml and deploy/static/starboard.yaml as well. See also my comment regarding the hack/update-starboard.yaml.sh script.